js/src/components/contact-information/phone-number-card/useCountdown.test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * External dependencies
+ */
+import { renderHook, act } from '@testing-library/react-hooks';
+
+/**
+ * Internal dependencies
+ */
+import useCountdown from './useCountdown';
+
+describe( 'useCountdown', () => {
+	it( 'initially should return `{ second: 0, callCount: 0, startCountdown: Function }`', () => {
+		const { result } = renderHook( () => useCountdown( 'A' ) );
+
+		expect( result.current.second ).toBe( 0 );
```

Comment:
:nail_care: 
Personally, I like to use [`toHaveProperty` matchers](https://jestjs.io/docs/expect#tohavepropertykeypath-value) in such cases to get more descriptive failure messages if somethin gets off
```suggestion
		expect( result.current ).toHaveProperty( 'second', 0 );
```
```

Reply:
💅  Personally I would prefer to use static dot notation `.second` here, especially when `second` is expected to be always there. I'd use `toHaveProperty` for properties that would be dynamically added and removed at run time.

Reply:
But we are testing here `useCountdown` and that's `useCountdown` that adds `.second`, so without the tested function, it's not always there.

Now, consider someone, unaware that `second` is expected by the outside world, changes it internally to `.remainingSeconds` thinking it's just an implementation detail.

Then, we would get 
```
  ● useCountdown › initially should return `{ second: 0, callCount: 0, startCountdown: Function }`

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: undefined
```

instead of 
```
  ● useCountdown › initially should return `{ second: 0, callCount: 0, startCountdown: Function }`

    expect(received).toHaveProperty(path, value)

    Expected path: "second"
    Received path: []

    Expected value: 0
    Received value: {"callCount": 0, "remainingSeconds": 0, "startCountdown": [Function anonymous]}

```

I agree, it's not a big difference. The real thing would be if `current` would change 
```
  ● useCountdown › initially should return `{ second: 0, callCount: 0, startCountdown: Function }`

    TypeError: Cannot read property 'second' of undefined
```
(but that's somewhat assured by `useRef` => `React`)

Reply:
I think the `current` is assured by the `renderHook`. We don't have to worry whether it would be changed.

Not sure if you get the same output as I do, the place below of it indicates which assertion is failed by a `^` symbol, and the content should be descriptive enough.

```
  ● useCountdown › initially should return `{ second: 0, callCount: 0, startCountdown: Function }`

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: undefined

      13 | 		const { result } = renderHook( () => useCountdown( 'A' ) );
      14 |
    > 15 | 		expect( result.current.second ).toBe( 0 );
         | 		                                ^
      16 | 		expect( result.current.callCount ).toBe( 0 );
```